### PR TITLE
Wqp 1607 partitioning

### PR DESCRIPTION
**Pull request by ee-usgs**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,28 +7,36 @@
...
-    <ciManagement>
-        <system>Jenkins</system>
-        <url>https://jenkins.wma.usgs.chs.ead/view/WQP/job/etl-stewards-build/</url>
-    </ciManagement>
-
     <scm>
-        <connection>scm:git:https://github.com/NWQMC/etl-stewards.git</connection>
-        <developerConnection>scm:git:git@github.com:NWQMC/etl-stewards.git</developerConnection>
-        <url>https://github.com/NWQMC/etl-stewards.git</url>
+        <connection>scm:git:https://github.com/NWQMC/etl-epa.git</connection>
```
Comment (orosz-usgs) [inline pom.xml:14]: Shouldn't this still point to the etl-stewards git repository?
not sure what this is used for.

File: pom.xml [hunk 2]
```diff
@@ -7,28 +7,36 @@
...
+        <developerConnection>scm:git:git@github.com:NWQMC/etl-epa.git</developerConnection>
+        <url>https://github.com/NWQMC/etl-epa.git</url>
```
Comment (orosz-usgs) [inline pom.xml:16]: same as above.

File: pom.xml [hunk 3]
```diff
@@ -39,7 +47,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.3.4.RELEASE</version>
+        <version>2.3.5.RELEASE</version>
```
Comment (orosz-usgs) [inline pom.xml:50]: good catch.

File: src/test/java/gov/acwi/wqp/etl/EtlStewardsIT.java [hunk 4]
```diff
@@ -121,6 +133,8 @@
 			query=UpdateLastETLIT.EXPECTED_DATABASE_QUERY_LAST_ETL)
 
 	public void endToEndTest() {
+
+		System.out.println("EXPECTED_DATABASE_QUERY_TABLE: " + EXPECTED_DATABASE_QUERY_TABLE);
```
Comment (orosz-usgs) [inline src/test/java/gov/acwi/wqp/etl/EtlStewardsIT.java:137]: Could be deleted.

File: pom.xml (hunk 1 above)
Comment (ee-usgs) [reply pom.xml:14]: Fixed this in the other pom-related merge request.

File: pom.xml (hunk 2 above)
Comment (ee-usgs) [reply pom.xml]: fixed in separate pom update

**Pull request merged**